app/queries/linked_items_query.rb

```diff
@@ -0,0 +1,84 @@
+class LinkedItemsQuery
+  attr_reader :content_item
+
+  def initialize(content_item)
+    @content_item = content_item
+  end
+
+  def call
+    items = linked_items
+    items["available_translations"] = available_translations if available_translations.any?
+    if content_item.format == "working_group"
+      items["policies"] = content_items_by_id.values.select { |content_item| content_item.format == "policy" }
```

Comment:
As for the overwriting issue, I would put the links in a different key in the content item:

Something like:

```
{
  content_id: "item-b",
  links: { ... },
  incoming_links: { policies: ["item-a"] }
}
```